static/index.html

```diff
@@ -1,74 +1,81 @@
-          <span class="status-log"></span>
+      </div>
+    </div>
+
+    <div id="success-modal" class="success-modal">
+      <div class="success-modal-content">
+        <div class="success-modal-header">
+          <h1 class="success-title">Success</h1>
+          <button class="success-close-modal">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 -960 960 960" fill="#999">
+              <path d="m336-280-56-56 144-144-144-143 56-56 144 144 143-144 56 56-144 143 144 144-56 56-143-144-144 144Z"></path>
+            </svg>
+          </button>
         </div>
-        <div class="mb-3 pos-rel">
-          <label for="outKey" class="form-label">OUTPUT</label>
-          <div class="out-file">ubiquibot-config.yml</div>
-          <textarea type="text" class="form-control" id="outKey" disabled="true"></textarea>
+        <div class="success-modal-body">
+          <div id="success-message" class="success-message"></div>
         </div>
       </div>
-      <div id="step2" class="step-container hidden">
-        <div class="mb-3">
-          <label for="allowance" class="form-label">ALLOWANCE AMOUNT</label>
-          <div>
-            <input type="text" class="form-control" style="max-width: 200px" id="allowance" />
-            <span style="font-size: 1.2rem; color: #808080">DAI</span>
-            <span class="status-log"></span>
+    </div>
+
+    <div id="content-area">
+      <div class="approve">
+        <div class="approve-header">
+          <h1>Approve Funding</h1>
+          <p>The selected amount in the selected chain will be approved as budget to honor issue rewards.</p>
+        </div>
+        <div class="approve-content">
+          <div class="current-allowance">
+            <span>Current allowance:</span>
+            <span class="current-allowance-amount">-</span>
+          </div>
+          <div class="approve-selectors">
+            <w3m-network-button></w3m-network-button>
+            <input class="amount-selector" type="text" name="amount" min="0" inputmode="numeric" placeholder="Enter amount" />
+            <input class="token-selector" list="token-options" type="text" name="address" placeholder="Enter token address"/>
+            <datalist id="token-options"></datalist>
           </div>
         </div>
-      </div>
-
-      <div class="mb-3">
-        <label for="statusKey" class="form-label">STATUS</label>
-        <input type="text" class="form-control" id="statusKey" disabled="true" />
-      </div>
-      <div class="mb-3 loader-wrap" style="display: none">
-        <span class="loader-x"></span>
-      </div>
-      <div class="btn-container">
-        <button type="button" class="btn btn-primary mb-3" id="setBtn">🌐 Set</button>
-      </div>
+        <div class="approve-footer">
+          <button disabled class="approve-button">Approve</button>
+          <button disabled class="revoke-button">Revoke Approval</button>
+        </div>
     </div>
```

Comment:
```suggestion
      </div>
    </div>
```

`<div class="approve">` is missing a closing `</div>`